src/components/Dialog/DialogStake.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <v-dialog :value="show" width="440" persistent>
+    <v-card>
+      <div align="right">
+        <v-btn icon @click="closeDialog" class="pr-2 pt-2">
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </div>
+      <div class="px-5">
+        <div align="center" class="pt-5 pb-8">
+          <strong>Stake Onboarding</strong>
+        </div>
+        <div class="pb-5 px-5">
+          <ol>
+            <li class="pb-5">
+              You need to stake DBIO to activate your account. There’s no
+              locking period. Your fund can be unstaked anytime with your
+              consent but there will be a period of 6 days to process it.
+            </li>
+            <li class="pb-5">
+              By Unstaking your fund, you will be an inactive users.
+            </li>
+            <li class="pb-5">
+              On the condition that your verification is rejected, your fund
+              will be refunded.
+            </li>
+          </ol>
+        </div>
+        <div class="d-flex justify-space-between mb-3">
+          <div><strong>Staking Amount</strong></div>
+          <div><strong>50,000 DBIO</strong></div>
```

Comment:
this part should be dynamic and to set/get the amount itself is from debio-node : query `labs.minimumStakeAmount()` . if it returns `none` then the default is 50k DBIO